main/src/modules/abm/abmReasoning/src/abmReasoning.cpp

```diff
@@ -186,6 +186,8 @@ bool abmReasoning::respond(const yarp::os::Bottle& bCommand, yarp::os::Bottle& b
 {
 
     bReply.clear();
+    bReply.addVocab(Vocab::encode("many"));
```

Comment:
It is used to display the help message not in one big line. Accessory and mainly for debug use. Can be removed.